katran/lib/testing/katran_tester.cpp

```diff
@@ -27,6 +27,7 @@
 
 #include "KatranOptionalTestFixtures.h"
 #include "KatranTestFixtures.h"
+#include "KatranGueTestFixtures.h"
```

Comment:
Our linter suggests :)
+#include "katran/lib/testing/KatranGueTestFixtures.h"

Reply:
sep diff. all of the headers has relative path